docs/content/cloudinary.md

```diff
@@ -0,0 +1,242 @@
+---
+title: "Cloudinary"
+description: "Rclone docs for Cloudinary backend"
+versionIntroduced: "v1.0"
```

Comment:
This would need to be v1.69.0 (I don't think we'll make the v1.68 release which is coming up in a few days

Reply:
Done